src/uct/api/uct.h

```diff
@@ -773,18 +773,18 @@ void uct_worker_progress(uct_worker_h worker);
  * @ingroup UCT_CONTEXT
  * @brief Add a slow path callback function to a worker progress.
  *
- * If 'id_p' points to a NULL identifier, this function will add a callback which
- * will be called every time a progress is made on the worker.
+ * If 'id_p' points to UCS_CALLBACKQ_ID_NULL, this function will add a callback
+ * which will be called every time a progress is made on the worker.
  *
  * @param [in]    worker        Handle to the worker whose progress should invoke
  *                              the callback.
  * @param [in]    func          Pointer to the callback function.
  * @param [in]    arg           Argument for the callback function.
  * @param [in]    flags         Callback flags, see @ref ucs_callbackq_flags.
  * @param [inout] id_p          Filled with callback identifier, which can be
- *                              used to remove the callback. If *id_p is non-NULL,
- *                              no operation will be performed and *id_p will be
- *                              left unchanged.
+ *                              used to remove the callback. If *id_p is not equal
+ *                              to UCS_CALLBACKQ_ID_NULL, no operation will be
+ *                              performed and *id_p will be left unchanged.
```

Comment:
More suggested re-wording:

 @param [inout] id_p          Points to a location to store a callback identifier. If *id_p is equal to
                              UCS_CALLBACKQ_ID_NULL, a callback will be added and *id_p will be replaced with a
                              callback identifier which can be subsequently used to remove the callback. Otherwise,
                              no callback will be added and *id_p will be left unchanged.